<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: white;
background-color: #121212;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_mult">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>210</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_binary</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btn_sqrt">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>310</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>√</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_graph">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(255, 85, 0);</string>
    </property>
    <property name="text">
     <string>нарисовать график</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_pow">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>390</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>^</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_binary</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btn_minus">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>260</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_binary</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btn3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>230</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_digits</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btn4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>280</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_digits</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btn_cos_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>310</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_delet">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn7">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>330</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_digits</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btn1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>230</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_digits</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btn_pi">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>350</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Pi</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clear">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>430</y>
      <width>531</width>
      <height>101</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>141</width>
      <height>101</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>101</height>
     </size>
    </property>
    <property name="whatsThis">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0);
font: 48pt &quot;Rockwell&quot;;</string>
    </property>
    <property name="text">
     <string>C*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cos">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>270</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>230</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_digits</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>501</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 48pt &quot;Rockwell&quot;;
color: rgb(0, 255, 237);</string>
    </property>
    <property name="text">
     <string>Kaml calculation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_div">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>160</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_binary</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btn8">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>330</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_digits</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btn6">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>280</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_digits</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btn_eq">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>370</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 0);</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_square">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>390</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>^2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn0">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_digits</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btn5">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>280</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_digits</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btn_dot">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>370</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>,</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sin">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>270</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn9">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>330</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_digits</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btn_plus">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>310</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_binary</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btn_fact">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>350</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>!</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="table">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>501</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 255, 237);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>430</y>
      <width>51</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 5pt &quot;.AppleSystemUIFont&quot;;</string>
    </property>
    <property name="text">
     <string>вывести прикол</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>509</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_digits"/>
  <buttongroup name="buttonGroup_binary"/>
 </buttongroups>
</ui>
